### PR TITLE
remove -- and fix alignment for edit tokens

### PR DESCRIPTION
**Pull request by MillerApps**
Removed the -- on the token page as well as fixed alignment of ETN label on the edit tokens section.

### CODE REVIEW

File: Trust/Tokens/Views/EditTokenTableViewCell.swift [hunk 1]
```diff
@@ -22,7 +22,12 @@ class EditTokenTableViewCell: UITableViewCell {
             tokenLabel.font = viewModel.titleFont
             tokenLabel.textColor = viewModel.titleTextColor
             tokenEnableSwitch.isOn = viewModel.isEnabled
-            tokenContractLabel.text = viewModel.contractText
+            if viewModel.contractText == nil {
+                tokenContractLabel.isHidden = true
+                tokenContractLabel.removeFromSuperview()
```
Comment (michaelScoff) [inline Trust/Tokens/Views/EditTokenTableViewCell.swift]: I don't think it's a good idea to remove view. Can you use Stack View instead? it supposed to center the view if label is empty

Comment (michaelScoff) [inline Trust/Tokens/Views/EditTokenTableViewCell.swift]: I believe stack view will remove view if it's empty automatically

**Comment by michaelScoff** [comment]
@MillerApps I think initially we need to center labels on tokens view right?

**Comment by michaelScoff** [comment]
@MillerApps do you mind providing screenshot to see what changed?

**Comment by MillerApps** [comment]
@michaelScoff I guess my eyes are a bit off as they looked like they were centered.

**Comment by MillerApps** [comment]
simulator screen shot - iphone 8 plus - 2018-02-02 at 15 03 47
simulator screen shot - iphone 8 plus - 2018-02-02 at 15 03 41

**Comment by michaelScoff** [comment]
I believe Ethereum (ETH) is not centered, you can check by enabling Debug View Hierarchy 

is there a way not to use `removeFromSuperview` call ?

**Comment by MillerApps** [comment]
@michaelScoff Yes, it works without removeFromSuperView.

File: Trust/Tokens/Views/EditTokenTableViewCell.swift [hunk 2]
```diff
@@ -22,7 +22,12 @@ class EditTokenTableViewCell: UITableViewCell {
             tokenLabel.font = viewModel.titleFont
             tokenLabel.textColor = viewModel.titleTextColor
             tokenEnableSwitch.isOn = viewModel.isEnabled
-            tokenContractLabel.text = viewModel.contractText
+            if viewModel.contractText == nil {
+                tokenContractLabel.isHidden = true
```
Comment (michaelScoff) [inline Trust/Tokens/Views/EditTokenTableViewCell.swift]: can you move `isTokenContractLabelHidden` into view model? Cell does not need to do any logic in this case

**Comment by michaelScoff** [comment]
@MillerApps just double check alignment in debug view and this should be good to go! + previous comment

File: Trust/Tokens/Views/EditTokenTableViewCell.swift (hunk 2 above)
Comment (MillerApps) [reply Trust/Tokens/Views/EditTokenTableViewCell.swift]: I can when I get home. I definitely feel a bit rusty, haven't done much iOS work in a good while.

**Comment by michaelScoff** [comment]
@MillerApps since you are doing this change can you set line break for the contract in the middle?

**Comment by MillerApps** [comment]
@michaelScoff I checked the alignment it's a hair off, I'll have to tweak it.

**Comment by MillerApps** [comment]
Yes when I get back home.

**Comment by michaelScoff** [comment]
@MillerApps nice catch! Making app pixel perfect is hard, but we gonna make it!

**Comment by MillerApps** [comment]
Should be all set now.

**Comment by MillerApps** [comment]
FYI Travis gets hung up on installing TrezorCrypto.

**Comment by michaelScoff** [comment]
@MillerApps Nice work! Merging this now!

@aleph7 can you look into Trezor issue?

**Pull request merged**